lib/embulk/input/twitter_ads_analytics.rb

```diff
@@ -280,6 +317,20 @@ def entity_plural(entity)
           "FUNDING_INSTRUMENTS"
         end
       end
+
+      private
+
+      def exp_backoff_sec(response:, retries:)
+        rate_limit_reset_timestamp = get_rate_limit_reset_timestamp(response: response)
+        (rate_limit_reset_timestamp.presence || (Time.zone.now + retries.second)).to_i - Time.zone.now.to_i
+      end
+
+      # https://developer.twitter.com/ja/docs/twitter-ads-api/rate-limiting
+      def get_rate_limit_reset_timestamp(response:)
+        # It is stored in the header in the format key => [value].
+        # @example x-rate-limit-reset"=>["1638316658"]
```

Comment:
`x-rate-limit-reset` returns a unix timestamp?
In that case, we should convert to seconds for wait.